### ISSUE TITLE
importing bookmarks

### ISSUE THREAD

**Issue by PAL-Thailand** [Feodor2/Mypal68#259]
Hello, Feodor2

I have just downloaded and installed MyPAL.  I have also exported my Firefox bookmarks into an HTML file.

I need some help from in getting bookmarks imported into MyPal as I am a truly new member of the MyPal browser.  Looking forward to your assistance.

Pal in Thailand

**Comment by PAL-Thailand** [comment]
I'm not having any success in attaching a screenshot of my work-to-date by using the "Attach files by dragging & dropping, selecting or pasting them." shown in the post. Please advise if GitHub has alternative methods to receive screenshots

**Comment by Feodor2** [comment]
So what do you want import boorkmark or attach files. Please do one single issue for one single problem but not about reporting sites that do not work - there one big issue for this

**Comment by PAL-Thailand** [comment]
Hello Feodor2,

Thank you for your reply and advice regarding the reporting of site problems.

Being a Git Hub newbie, I am still very much in a learning phase as how to 
proceed with various issues and, thanks to you, I will be better informed on 
how and where to post different types of problems/situations that require some 
input from forum members or the powers that be at Git Hub. I will seek out the 
proper Git Hub page for reporting that the site's page for posting comments 
has a non-functional method for attaching files to a post isn't functioning as 
one would hope.

Now, in reply to your question, my answer is that I want to import my Firefox 
bookmarks into the MyPal browser.

Since my last posting, I have found a way to send screenshots to the forum 
members (as the existing methods provided at posting time do not work) and I 
am happy to show you that I was able, at least, to make MyPal accept the Firefox 
Bookmarks.html file.  

To see what I have been able to do in MyPal's Library so far, please visit this 
URL:  https://i.imgur.com/9N6Oht5.jpg

It would seem that the prerequisites for actually importing the contents of 
the Firefox Bookmarks.html file into the MyPal browser are present. But what 
has NOT HAPPENED is that the content of the Firefox Bookmarks.html file did not 
get imported/transferred into MyPal!

There must be a set of keystrokes (similar to the "Ctrl+Shift+B" bluedragon-cairo 
sent me)  or some button to push on that I do not see or have no access to in 
order to complete the actual transfer of the bookmarks into MyPal browser.

I would greatly welcome your further advice as to how I should proceed to complete the actual transfer of the bookmarks into MyPal.

**Comment by PAL-Thailand** [comment]
I am happy to report that I have finally been able to import Firefox bookmarks via an html file into Mypal.  The steps involved in the process were not overly easy to guess at and were certainly not easily evident.

In order to assist new users in attaching files to their posts, it might be a good idea to prepare a clear and easily understandable (i.e. with screenshots) user manual either within Mypal itself or as a separate, downloadable file for easy access.

Many thanks to the members who have patiently helped me to bring my newly installed Mypal to life with all the previous bookmarks I had accumulated over the years.

Consequently, I will "Close issue" with a large THANK YOU!
